src/helpers/utils.ts

```diff
@@ -142,3 +143,8 @@ export function calcToSeconds(value, unit) {
   if (unit === 'h') return value * 60 * 60;
   if (unit === 'd') return value * 60 * 60 * 24;
 }
+
+export function setPageTitle(message, params = {}) {
+  const { t } = i18n.global;
+  document.title = t(message, params) + ' - Snapshot';
```

Comment:
I would remove this here
```suggestion
  document.title = t(message, params);
```